fast/stages/0-bootstrap/variables.tf

```diff
@@ -97,6 +97,14 @@ variable "custom_roles" {
   default     = {}
 }
 
+variable "factories_config" {
+  description = "Configuration for the organization policies factory."
+  type = object({
+    data_dir = optional(string, "data/org-policies")
```

Comment:
Rename the key data_dir to org_policies?

Reply:
I was trying to be consistent with other stages, but yes it makes sense and we might want to eventuallys peciali members of that variable for other stages too (eg `factories_config.subnets`)